src/commands/new/selector.ts

```diff
@@ -20,20 +21,27 @@ module.exports = {
     try {
       const options = args.parseOptions(SELECTOR_OPTIONS);
       const outputDirectory = args.getOutputDirectory();
+      const filters = args.getSelectorFilters();
```

Comment:
You may not even need a method for this:
```suggestion
      const filters = JSON.parse(parameters.options.filters) || [];
```